### DISCUSSION TITLE
Surveys

### DISCUSSION THREAD

**Discussion by jmduke** [buttondown/discussions#6] [Announcements]
## TL;DR

Buttondown will add support for simple, one-question surveys that can be inserted within the email (or on web). 

## What's the problem?

"I want to poll my subscribers" has been a long-standing request from users: to know what kind of content resonates, to get demographic information, etc.

## Why implement it?

I've pushed back on implementing surveys/polls for a while, for four reasons:

1. It's a fairly niche feature. I would peg _saturated_ usage of surveys, if perfectly implemented, will be at something like 5% of the userbase.
2. Like drip automation or A/B testing, there's a lot of _fractal_ product work involved in surveys; it's a product category unto itself.
3. It doesn't obviously add compounding impact to the rest of the feature set; there aren't a lot of parts of Buttondown that immediately become more valuable just because surveys exist.
4. It's not obvious to me that it's such a compelling feature that it changes people's purchase decision; I don't think there are many people who are currently using another tool that will start using Buttondown because Buttondown added surveys, nor are there people who have left Buttondown because it didn't have surveys.

All of that being said...I think I have a good mental model of how to implement surveys in a way that:

1. Integrates nicely with subscribers and the external-facing API
2. Is relatively welterweight so as to not draw too much resourcing or attention away from other parts of the app

Moreover, there...aren't a lot of great options. There are many nice form-builder tools out there, but the story of integrating them with something like Buttondown is kind of meh — you have to do subscriber reconciliation yourself, it's hard to pre-fill, etc. So I think I can provide value to the 80/20 case of people who want simple, low-effort introspection into their subscriber base. 

## What's the solution?

(The solution goes here once I've designed more on it.)

## Rough timeline

**Late February** for a private beta; **March** for GA.

## Open questions

(None, yet!)

## Out of scope 

- Multiple questions within a single survey. You'll likely be able to insert multiple survey questions within a single email, but I explicitly do not want to handle _building an entire survey product_ within Buttondown. Tools like [Reform](https://reform.app/) or [Tally](https://tally.so/) are better at that.
